README.md

```diff
@@ -200,40 +200,11 @@ School Year 2006–07](http://nces.ed.gov/ccd/pdf/pau061bgen.pdf)
  * [rspec repo](http://github.com/dchelimsky/rspec)
  * [rspec-rails repo](http://github.com/dchelimsky/rspec-rails)
 
-#### Cucumber
-
-
-#### Capybara
-
-You can customize your selenium drivers by editing @ ~/.capybara.rb @
-This file is sourced by @ ./features/support/local_config.rb @
-
-Here is a sample file which checks for an ENV param named @
-SELENIUM_CONFIG @
-
-
-    case ENV
-    when 'saucelabs-ie'
-        Capybara.server_port = ENV.to_i
-        Capybara.app_host = "http://app#{Capybara.server_port}.test.dev.concord.org"
-        selenium_remote :url => "http://ccdev:[aebecf9c-b426-44f8-9726-6eb747a7340e@ondemand.saucelabs.com](mailto:aebecf9c-b426-44f8-9726-6eb747a7340e@ondemand.saucelabs.com):80/wd/hub",
-        :desired_capabilities => Selenium::WebDriver::Remote::Capabilities.internet_explorer
-    when 'ff6'
-        puts " ----- using Firefox 6 profile -----"
-        Selenium::WebDriver::Firefox.path= '/usr/local/bin/firefox6'
-        Capybara.register_driver :selenium do |app|
-            Capybara::Selenium::Driver.new
-        end
-    when 'chrome'
-        puts "----- using Chrome profile -----"
-        Capybara.register_driver :selenium do |app|
-            Capybara::Selenium::Driver.new
-        end
-    else
-        # by default don't customize anything, this ought to keep the current
-        capybara tests running as before
-        #
-    end
+#### Cucumber / Capybara
```

Comment:
This is great.  I am testing it now on my local machine.

You mentioned that you had a workflow that included using Pry to debug. I think I could work it out, but could you please add it to this part of the README?  A more detailed set of instructions for a developer who is trying to get a cucumber test to pass would be helpful.

Reply:
Cool, thanks for trying it out @knowuh. 

@emilywatkins added some documentation about the use case for pry in non-headless mode: 

https://github.com/concord-consortium/rigse/pull/549/commits/189e950e59e931fd3192643b0de9943dfb5cf3e5

Let us know if you have any feedback regarding the readme additions.

Reply:
Thanks for the clear instructions @emilywatkins !  👍